package.json

```diff
@@ -13,6 +13,7 @@
   "dependencies": {
     "basement": "https://github.com/sanctuarycomputer/basement.git",
     "build": "^0.1.4",
-    "highlight.js": "^9.13.1"
+    "highlight.js": "^9.13.1",
+    "jquery": "^3.3.1"
```

Comment:
Still looks like jquery is required.

Reply:
Oh shoot, only removed the CDN forgot to remove the package